<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPet</class>
 <widget class="QWidget" name="SelectPet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>宠物编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="pet" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLabel" name="pri">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>31</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>精力</string>
    </property>
   </widget>
   <widget class="QLabel" name="pri_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>61</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>物攻</string>
    </property>
   </widget>
   <widget class="QLabel" name="pri_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>91</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>物抗</string>
    </property>
   </widget>
   <widget class="QLabel" name="pri_4">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>121</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>魔攻</string>
    </property>
   </widget>
   <widget class="QLabel" name="pri_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>151</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>魔抗</string>
    </property>
   </widget>
   <widget class="QLabel" name="pri_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>181</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hp">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>31</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ad">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>61</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>91</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ap">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>121</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dp">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>151</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>181</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="hp2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="nuli">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>努力值</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ad2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dd2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>90</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ap2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dp2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>150</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="speed2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>180</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="nuli_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nulizhi">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>51</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="adp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="ddp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="app">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="dpp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="speedp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="nop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>51</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="adm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="ddm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="apm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="dpm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="speedm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="xg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>平衡</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>固执</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>孤僻</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>调皮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>勇敢</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>保守</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>稳重</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>马虎</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>冷静</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>胆小</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>开朗</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>急躁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天真</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大胆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>淘气</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>悠闲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>无虑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>沉着</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>慎重</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>狂妄</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>温顺</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>插入宠物</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改宠物</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="skills">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>180</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="skill_descp">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="pokes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>171</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="poks">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="poke_img">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="attr">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="attr2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="skills_selected">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>已选技能</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>可选技能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_skill">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>360</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_skill">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存队伍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载队伍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始战斗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_poke">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除一个宠物</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
